### ISSUE TITLE
TargetInvocationException when running with Nova

### ISSUE THREAD

**Issue by Mackiovello** [Starcounter/Starcounter.Linq#68]
Running both the queries in README gives me this exception when running on Nova:

```
Unhandled Exception: System.TypeInitializationException: The type initializer for 'Starcounter.Linq.KnownMethods' threw an exception. ---> System.TypeLoadException: Could not load type 'Starcounter.Nova.ObjectExtensions' from assembly 'Starcounter.Nova, Version=0.11.0.0, Culture=neutral, PublicKeyToken=null'.
   --- End of inner exception stack trace ---
   at Starcounter.Linq.Visitors.RootVisitor`1.VisitMethodCall(MethodCallExpression node, QueryBuilder`1 state)
   at Starcounter.Linq.QueryContext`1.Execute[TResult](Expression expression)
   at Starcounter.Linq.QueryProvider.System.Linq.IQueryProvider.Execute[T](Expression expression)
   at System.Linq.Queryable.Count[TSource](IQueryable`1 source)
   at Reprod.Program.<>c.<Main>b__0_1() in /home/erlend/Desktop/Reprod/Program.cs:line 74
   at Starcounter.Nova.Database.Context.Run(Transaction txn, Action action)
   at Starcounter.Nova.Database.Context.Transact(Action action, TransactOptions options)
   at Starcounter.Nova.Database.TransactImpl.Transact(Action action, TransactOptions options)
   at Reprod.Program.Main(String[] args) in /home/erlend/Desktop/Reprod/Program.cs:line 67
```

The queries in question are:

```cs
var johns = DbLinq.Objects<Person>().Where(p => p.FirstName == "John1"); 
var peopleByNameQuery = DbLinq.CompileQuery((string name) =>
    DbLinq.Objects<Person>().Where(p => p.FirstName == name));
```

```xml
  <ItemGroup>
    <PackageReference Include="runtime.native.Starcounter.Bluestar2" Version="2.0.1" />
    <PackageReference Include="Starcounter.Linq" Version="3.2.0" />
    <PackageReference Include="Starcounter.Nova.Hosting" Version="0.11.0-*" />
  </ItemGroup>
```

**Comment by Mackiovello** [comment]
I think I might know what the problem is. I'll look at it tomorrow.

**Comment by Mackiovello** [comment]
The problem is that the released version of Starcounter.Linq, 3.2.0, doesn't work with Nova 0.11.0.

I tried with latest changes on the `nova` branch in Starcounter.Linq, and that worked.

@un-tone can you publish a new package for Starcounter.Linq?

**Comment by un-tone** [comment]
@Mackiovello released [3.3.0](https://github.com/Starcounter/Starcounter.Linq/releases/tag/3.3.0)

**Comment by Mackiovello** [comment]
> @Mackiovello released 3.3.0

Thanks! It works now
